package/tailscale-systemd/tailscaled.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Tailscale node agent
+Documentation=https://tailscale.com/kb/
+Wants=network-pre.target
+After=network-pre.target
+
+[Service]
+Environment="HOME=/home/root"
+ExecStartPre=/opt/bin/tailscaled --cleanup
+ExecStart=/opt/bin/tailscaled --state=/var/lib/tailscale/tailscaled.state --socket=/run/tailscale/tailscaled.sock --tun=userspace-networking --socks5-server=localhost:1055 --outbound-http-proxy-listen=localhost:1055
```

Comment:
Try to place things under `/opt` if possible. So `/var/lib/tailscale/tailscaled.state` should be `/opt/var/lib/tailscale/tailscaled.state`. it's fine to use `/run` as it's tmpfs.